filtra.go

```diff
@@ -3,38 +3,51 @@ package main
 import (
 	"flag"
 	"fmt"
+	"io"
+	"os"
 	"strings"
 	"time"
 
 	"database/sql"
+
 	"github.com/jasonlvhit/gocron"
 	_ "github.com/lib/pq"
 	log "github.com/sirupsen/logrus"
 )
 
-var debugFlag = flag.Bool("debug", false, "Sets log level to debug.")
-var configFileFlag = flag.String("config", "./config.toml", "Path to config file")
+const (
+	// exitFail is the exit code if the program
+	// fails.
+	exitFail = 1
```

Comment:
Will this be applicable to all the situations where the program exits due to a log.fatal call?

Reply:
`log.Fatal` also exits with 1, but is not tied to this in any way. The plan is to use and extend this in the future.